docs/RelayAPI.md

```diff
@@ -1,376 +1,504 @@
-
-`GET /networks/:networkAddress/users/:userAddress/trustlines`
-
-### Response
+GET /networks/:network/users
+```
+#### Parameters
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+#### Response
 ```javascript
 [
-  {
-    "address": "0xb33f5gaac...", // address of user B
-    "balance": "100", // balance of trustline from POV of user A
-    "given": "500", // credit line given by A
-    "leftGiven": "400" // given - balance
-    "received": "600", // credit line received from B
-    "leftReceived": "700" // received + balance
-    "interestRate": "0.1" // TODO PROBLEM: rounding in js
-  },
-  ... // other trustlines
+  "0xcbF1153F6e5AC01D363d432e24112e8aA56c55ce",
+  "0x7Ff66eb1A824FF9D1bB7e234a2d3B7A3b0345320",
+  "0x7Ec3543702FA8F2C7b2bD84C034aAc36C263cA8b",
+	...
 ]
 ```
 
-## Get trustline to user
-Return a trustline between A and B in a currency network if one exists.
-
-`GET /networks/:networkAddress/users/:userAddressA/trustlines/:userAddressB`
+---
 
-### Response
-```javascript
-{
-  "address": "0x...", // address of user B
-  "balance": "100", // balance of trustline from POV of user A
-  "given": "500", // credit line given by A
-  "received": "600", // credit line received from B
-  "interestRate": "0.1" // PROBLEM: rounding in js
-}
-```
-
-## Get total spendable amount
-Returns amount a user can spend in a currency network.
-
-`GET /networks/:networkAddress/users/:addressA/spendable`
-
-### Response
-```javascript
-1000
+### User details in currency network 
+### `/networks/:network/users/:user`
+Returns detailed information of an user in a currency network.
+#### Request
 ```
-
-## Get spendable amount to user
-Returns amount a user can spend in a currency network to another user.
-
-`GET /networks/:networkAddress/users/:addressA/spendables/:addressB`
-
-### Response
-```javascript
-90
+GET /networks/:network/users/:user
 ```
-
-## Get path
-Returns the cheapest path and maximal sendable amount with calculated fees if existent
-
-`GET /networks/:networkAddress/users/:aAddress/path/:bAddress?value=:value&maxHops=:maxHops&maxFees=:maxFees`
-
-### Parameters
-- `value` - amount of money that should be transferred through path (optional)
-- `maxHops` - specify maximal hops for path (optional)
-- `maxFees` - specify maximal fees user is ready to pay (optional)
-
-### Response
+#### Parameters
+|Name|Type|Required|Description|
+|-|-|-|-|
+|network|string|YES|Address of currency network|
+|user|string|YES|Address of user|
+#### Response
 ```javascript
 {
-    "path": ["0xabc123bb...", "0xeebc3bb...", ...], // addresses of users in path
-    "estimatedGas": "12000000",
-    "fees": "0.12" // fees for calculated path
+	"balance": -1000, // sum over balances of all trustlines user has in currency network
+  "given": 2000, // sum of all creditlines given by user in currency network
+  "received": 3000, // sum of all creditlines received by user in currency network
```

Comment:
Are that actually numbers and not strings? Then this is a big. This should be strings like in the events.

Reply:
Yes currently numbers are returned. I will open an issue for this.